### PR TITLE
Limits to required labels when querying for App

### PR DESCRIPTION
**Pull request by cyriltovena**

### CODE REVIEW

File: public/app/overrides/services/apps.ts [hunk 1]
```diff
@@ -80,11 +80,11 @@ function removeDuplicateApps(app: App[]) {
 export async function fetchApps(): Promise<
   Result<App[], RequestError | ZodError>
 > {
-  // TODO: is this the best query?
   const response = await requestWithOrgID('/querier.v1.QuerierService/Series', {
     method: 'POST',
     body: JSON.stringify({
       matchers: [],
+      labelNames: [PyroscopeAppLabel, ServiceNameLabel, '__profile_type__'],
```
[the remark this answers is not in the document]
Comment (cyriltovena) [reply public/app/overrides/services/apps.ts:87]: ho that sux

[the remark this answers is not in the document]
Comment (simonswine) [reply public/app/overrides/services/apps.ts:87]: not too sure maybe something we can change in connect-go

**Review by Rperry2174 — APPROVED** [review]
Lgtm

**Comment by cyriltovena** [comment]
It works for both, looking at adding service reflection and next time we should bump the service version for that type of change

>Many gRPC-specific tools depend on server reflection, which lets callers access your service's Protobuf schema at runtime. Connect supports server reflection with the github.com/bufbuild/connect-grpcreflect-go package. Keep in mind that there are two versions of the gRPC server reflection API, and many tools (including grpcurl) still use the older one — most services should mount handlers from both grpcreflect.NewHandlerV1 and grpcreflect.NewHandlerV1Alpha.

**Pull request merged**